docs/release-notes/version-2.6.md

```diff
@@ -2,6 +2,7 @@
 
 ## Enhancements
 
+* [#2365](https://github.com/netbox-community/netbox/issues/2365) - Toggle for showing available prefixes/ip addresses
```

Comment:
I closed #2365 as a duplicate in favor of #2598. If you can tweak this great, otherwise I'll fix it during merge.

Reply:
Sure. I'll fix it along with the template thing.